src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/Config/AddAzureRmVMRunCommandParameterCommand.cs

```diff
@@ -0,0 +1,80 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using Microsoft.Azure.Commands.Compute.Automation.Models;
+using Microsoft.Azure.Management.Compute.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Compute.Automation
+{
+    [Cmdlet("Add", "AzureRmVMRunCommandParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(PSRunCommandInput))]
+    public partial class AddAzureRmVMRunCommandParameterCommand : Microsoft.Azure.Commands.ResourceManager.Common.AzureRMCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            ValueFromPipelineByPropertyName = true)]
+        public PSRunCommandInput RunCommandInput { get; set; }
```

Comment:
@hyonholee I was thinking this parameter should be renamed to `InputObject` with an alias to `RunCommandInput`, which will follow the recommend conventions from the PowerShell Core team, but I'm not sure if `InputObject` would be confusing since it shares the `Input` noun with the `RunCommandInput` type. What do you think?